### ISSUE TITLE
Reorganize Content Security Policy Article for Implementation

### ISSUE THREAD

**Issue by jpmedley** [google/WebFundamentals#3491]
The current version is expository and give little implementation advice.

**Comment by PaulKinlan** [comment]
If this is the case, then would move this into the primers section at this point....

**Comment by addyosmani** [comment]
Primers sgtm

**Comment by petele** [comment]
I don't think it should move into primers. Primers should be reserved for building blocks, things that on their own aren't very interesting, but when used across different pieces, become very interesting. CSP isn't a building block, but security feature.

**Comment by petele** [comment]
Can you add some detail on how this is different than #3490?

**Comment by jabzzy** [comment]
Hope, that this is a proper place for this kind of question: this article is due to some restructuring since 2016. Is it going to be updated? It, probably, should, since it contains some misleading info, e.g., in the "Implementation details" section — providing sth like `*://*.example com` as a directive value would cause it to be skipped with this kind of error message (as tested in Chromium 60) — ![message](https://uploads.disquscdn.com/images/7a0acff744a356d685301532a779f9f0ef37a6c3a9c35caf5c479edf199b7ebc.png?w=800&h=32)
